src/Webapi/Webapi__Dom.re

```diff
@@ -71,6 +72,7 @@ include Webapi__Dom__Types;
 [@bs.val] external document : Dom.document = "document";
 [@bs.val] [@bs.scope "window"] external history : Dom.history = "history";
 [@bs.val] [@bs.scope "window"] external location : Dom.location = "location";
+[@bs.val] [@bs.scope "window"] external navigator : Webapi__Dom__Navigator.t = "navigator";
```

Comment:
I realize there are similar bindings above it, but I really don't think `Webapi.Dom.navigator` is a sensible place to access a `Navigator` object from. Since the `Webapi.Dom.Window.navigator` binding already exists, we don't need another one here.